### PR TITLE
Rename Recipe DSL to Chef Infra Language

### PR DESCRIPTION
**Pull request by tas50**
That's what this has always been and then we don't have to overload folks with CS theory and explain what a DSL is.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by IanMadd** [comment]
All the changes made to files in the `_vendor` directory should be made in their source repos and then we can update them here.

File: _vendor/github.com/inspec/inspec/docs-chef-io/content/inspec/resources/os.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ Use `os.family` to enable more granular testing of platforms, platform names, ar
 - `bsd` For platforms that are part of the Berkeley OS family `darwin`, `freebsd`, `netbsd`, and `openbsd`.
 - `debian`
 - `hpux`
-- `linux`. For platforms that are part of the Linux family `alpine`, `amazon`, `arch`, `coreos`, `exherbo`, `fedora`, `gentoo`, and `slackware`.
+`linux`. For platforms that are part of the Linux family `alpine`, `amazon`, `arch`, `coreos`, `exherbo`, `fedora`, `gentoo`, and `slackware`.
```
Comment (IanMadd) [inline _vendor/github.com/inspec/inspec/docs-chef-io/content/inspec/resources/os.md]: This should get yanked too and done in `inspec/inspec`

Comment (tas50) [reply _vendor/github.com/inspec/inspec/docs-chef-io/content/inspec/resources/os.md]: I clearly messed up reverting that. Let me fully back it out.

File: content/glossary.md [hunk 2]
```diff
@@ -14,17 +14,6 @@ aliases = ["/glossary.html"]
     weight = 70
 +++
 
-**Acceptance**
```
Comment (kagarmoe) [inline content/glossary.md]: Can we break these Workflow related changes out into another PR? I need to make sure we've got them all.

File: content/infra_language.md [hunk 3]
```diff
@@ -1,48 +1,37 @@
...
-{{% dsl_recipe_summary %}}
+{{% infra_lang_summary %}}
 
-Because the Recipe DSL is a Ruby DSL, anything that can be done using
-Ruby can also be done in a recipe or custom resource, including `if` and
-`case` statements, using the `include?` Ruby method, including recipes
-in recipes, and checking for dependencies. See the [Ruby
-Guide](/ruby/) for further information on built-in Ruby
-functionality.
+The Chef Infra Language is based on Ruby, allowing you to utilize the power of Ruby when the built-in language doesn't meet your needs out of the box. If you'd like to learn more about extending your Chef Infra code by using Ruby see our [Ruby Guide](/ruby/) for further information on Ruby functionality.
```
Comment (kagarmoe) [inline content/infra_language.md]: Utilize -> use

Comment (kagarmoe) [reply content/infra_language.md]:
```suggestion
The Chef Infra Language is based on Ruby, giving you access to the power of Ruby when the built-in language doesn't meet your needs. Learn more about extending your Chef Infra code in our [Ruby Guide](/ruby/).
```

File: content/infra_language.md [hunk 4]
```diff
@@ -1,48 +1,37 @@
...
 
-## Include Recipes
+## Resources
 
-{{% cookbooks_recipe_include_in_recipe %}}
-
-### Reload Attributes
+Resources are the cornerstone of the Chef Infra Language. Resources define the desired state of an object on a system. A resource can be as simple as a directory or as complex or a complete security policy. Chef Infra Client ships with over 150 resources for configuring components such as packages, file, directories, or firewalls. For more information on resources in Chef Infra Client including a complete list of those included out of the box see [Resources](/resources).
```
Comment (kagarmoe) [inline content/infra_language.md]: Can you give me something more specific term (or set of terms) than "object" in the "desired state of an object on a system" 

```suggestion
The Chef Infra Language expresses the desired state of configurable objects in your system in resources. They are the building blocks of recipes and they are central to understanding configuration management with Chef Infra. Some resources are as simple as a directory name, while others as complex as a complete security policy. Chef Infra Client ships with over 150 pre-packaged resources for configuring components such as packages, files, directories, and firewalls. Learn more in the Chef Infra Client [Resources](/resources) documentation.
```

File: themes/docs-new/layouts/shortcodes/infra_lang_summary.md [hunk 5]
```diff
@@ -0,0 +1 @@
+The Chef Infra Language is a comprehensive systems configuration language with resources and helpers for configuring various operating systems. The language is primarily used in Chef Infra recipes and custom resources to tell the Chef Infra Client what action(s) to take to properly configure a system. The Chef Infra Language provides resources for system-level components such as packages, users, or firewalls, and also includes helpers to allow you to make configuration decisions based on operating systems, clouds, virtualization hypervisors, and more.
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/infra_lang_summary.md]:
```suggestion
The Chef Infra Language is a comprehensive systems configuration language with resources and helpers for configuring operating systems. The language is primarily used in Chef Infra recipes and custom resources to tell the Chef Infra Client what action(s) to take to configure a system. The Chef Infra Language provides resources for system-level components such as packages, users, or firewalls, and it also includes helpers to allow you to make configuration decisions based on operating systems, clouds, virtualization hypervisors, and more.
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: content/glossary.md (hunk 2 above)
Comment (tas50) [reply content/glossary.md]: Should I leave the ones that aren't workflow. There's some really old stuff in here from Analytics and Reporting

Comment (kagarmoe) [reply content/glossary.md]: We can pull out the Analytics and Reporting terms, too. We'll use this new PR as the basis to sweep the docs for related content that we need to pull.

File: content/infra_language.md [hunk 6]
```diff
@@ -16,11 +16,11 @@ aliases = ["/dsl_recipe.html", "/dsl_recipe"]
...
 
 ## Resources
 
-Resources are the cornerstone of the Chef Infra Language. Resources define the desired state of an object on a system. A resource can be as simple as a directory or as complex or a complete security policy. Chef Infra Client ships with over 150 resources for configuring components such as packages, file, directories, or firewalls. For more information on resources in Chef Infra Client including a complete list of those included out of the box see [Resources](/resources).
+The Chef Infra Language expresses the desired state of configurable objects in your system in resources. They are the building blocks of recipes and they are central to understanding configuration management with Chef Infra. Some resources are as simple as a directory name, while others as complex as a complete security policy. Chef Infra Client ships with over 150 pre-packaged resources for configuring components such as packages, files, directories, and firewalls. Learn more in the Chef Infra Client [Resources](/resources) documentation.
```
Comment (kagarmoe) [inline content/infra_language.md]:
```suggestion
The Chef Infra Language expresses the desired state of configurable objects in your system in resources. Resources are the building blocks of recipes and they are central to understanding configuration management with Chef Infra. Some resources are as simple as a directory name, while others as complex as a complete security policy. Chef Infra Client ships with over 150 pre-packaged resources for configuring components such as packages, files, directories, and firewalls. Learn more in the Chef Infra Client [Resources](/resources) documentation.
```

Comment (kagarmoe) [reply content/infra_language.md]: Unclear antecedent

**Review by kagarmoe — CHANGES_REQUESTED** [review]
One last nit (word change)

File: content/glossary.md (hunk 2 above)
Comment (tas50) [reply content/glossary.md]: Nuked from this PR

**Review by kagarmoe — APPROVED** [review]
tenor-93029711

**Pull request merged**